text/0233-opentelemetry-sandbox.md

```diff
@@ -0,0 +1,89 @@
+3. Proponents should abide by OpenTelemetry’s Code of Conduct (currently the same as CNCF’s).
+4. There’s no expectation that small sandbox projects will have regular calls, but there is an expectation that all decisions will be made in public and transparently.
+5. Sandbox projects do NOT have the right to feature OpenTelemetry’s name on their websites.
+
+#### Template
+
+> Project name:
+>
+> Repository name:
+>
+> Motivation:
+>
+> Zoom room requested?
+
+##### Example
+
+> Project name: OpenTelemetry Collector Community Distributions
+>
+> Repository name: opentelemetry-collector-distributions
+>
+> Motivation: The OpenTelemetry Collector Builder allows people to create their own distributions, and while the OpenTelemetry Collector project has no intentions (yet) on hosting other more specialized distributions, some community members are interested in providing those distributions, along with best practices on building and managing such distributions, especially around the CI/CD requirements.
+>
+> Zoom room requested? No
+
+### Periodic reports
+
+On the first [GC/TC joint call](https://docs.google.com/document/d/1jylE5uZCKV9mrPw8Qrc5ExGyRVbBdqcbWPwni-hB5dE) of the calendar quarter (Jan, Apr, Jul, Oct), the TC/GC sponsors for the sandbox projects MUST provide an update about their sponsored project. Should the sponsor not be able to join the call, a short written report MUST be provided as part of the meeting notes.
+
+### Exiting the sandbox
+
+Projects can be hosted for at most 12 months and are expected to exit before that, with one of the following outcomes:
+
+- incorporation as part of an existing SIG
+- acceptance as a new SIG
+- archival
+
+When the sandbox project cannot get accepted as its own SIG nor incorporated as part of an existing SIG, it's automatically archived after 12 months.
+
+### Further details
+
+* A new GitHub user group will be created with the current members of the TC and GC as members. This group shall be the admin for all repositories in the organization.
+* Project proponents are added as maintainers and encouraged to recruit other maintainers from the community.
+* Code hosted under this organization is owned by the OpenTelemetry project and is under the governance of OTel’s Governance Committee.
+
+## Trade-offs and mitigations
+
+None.
+
+## Prior art and alternatives
+
+* One obvious alternative is to let users collaborate on their own accounts and organizations, as some are doing today. There are two problems with that: the first is that those projects usually lack clear governance rules, so that external contributors aren't sure what's going to happen with their code contributions. The second problem is lack of visibility: most of the current experiments and initiatives aren't listed in the [OpenTelemetry's registry](https://opentelemetry.io/ecosystem/registry/), and being under an OpenTelemetry organization would make those projects more visible while still at early stages.
```

Comment:
> those projects usually lack clear governance rules

Nothing prevents those projects from adopting the same license and code of conduct as CNCF/OTEL.

> lack of visibility

This proposal in no way addresses the visibility problem. If the attempt is to address is by simply being in the otel-sandbox list of repos (which isn't very search friendly), then there are much easier ways of achieving the same (e.g. a page / registry on the website with minimum entry requirements) than having a sandbox program.

Reply:
Being part of the OTel repository certainly makes it more visible: a blog post mentioning "github.com/open-telemetry-sandbox/some-experiment" would certainly get more attention than "github.com/jpkrohling/some-experiment". Other than that, you are right that there are no active steps being taken to make those repos more visible.